pkg/controllers/cstorvolumeconfig/start.go

```diff
@@ -33,10 +29,12 @@ import (
 	server "github.com/openebs/cstor-operators/pkg/server"
 	cvcserver "github.com/openebs/cstor-operators/pkg/server/cstorvolumeconfig"
 	"github.com/openebs/cstor-operators/pkg/snapshot"
+	"github.com/pkg/errors"
 	kubeinformers "k8s.io/client-go/informers"
 	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/rest"
 	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/klog"
 )
 
```

Comment:
Can we add a `klog.fatal` if [bind address](https://github.com/openebs/cstor-operators/blob/a08d9ae28665faf6e2e8416f7aa57750e54d5d76/pkg/controllers/cstorvolumeconfig/start.go#L174) is empty? This crash is required in a case where user doesn't specify `--bind` argument then the server start on pods [localhost](https://github.com/openebs/cstor-operators/blob/a08d9ae28665faf6e2e8416f7aa57750e54d5d76/pkg/server/config.go#L70) then other services in the cluster can not access the back/restore service.

This is required now since we are changing the way of passing arguments earlier it is always guaranteed that the entry script will pass value but now we are passing from YAML so for the safer side let's add a check and fatal if the value doesn't provide.

Reply:
Although since is going to be set via flag , its going to be set always using downward api , but added the fatal as well.